packages/react-dom/src/events/__tests__/SyntheticEvent-test.js

```diff
@@ -18,63 +17,122 @@ describe('SyntheticEvent', () => {
   var createEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticEvent = require('events/SyntheticEvent');
     React = require('react');
     ReactDOM = require('react-dom');
     ReactTestUtils = require('react-dom/test-utils');
-
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget')(nativeEvent);
-      return SyntheticEvent.getPooled({}, '', nativeEvent, target);
+    createEvent = (eventType, nativeEvent) => {
+      const defaultNativeEvent = {
+        bubbles: true,
+        cancelable: true,
+        ...nativeEvent,
+      };
+      var event = new Event([eventType, defaultNativeEvent]);
```

Comment:
Why is there an array here? Shouldn't it be just `new Event(eventType, defaultNativeEvent)`?

It would be nice to rename `defaultNativeEvent` and `nativeEvent` to `eventInit` and `defaultEventInit`, to match how spec calls these arguments.

Reply:
>Why is there an array here? Shouldn't it be just new Event(eventType, defaultNativeEvent)?

I was using arrays because I've seen the Event's constructor on `jsdom` and the first argument is `args` and it is an array: https://github.com/tmpvar/jsdom/blob/2d51af302581a57ee5b9b65595f1714d669b7ea2/lib/jsdom/living/events/Event-impl.js#L7

but no problem to change it :D

> It would be nice to rename defaultNativeEvent and nativeEvent to eventInit and defaultEventInit, to match how spec calls these arguments.

Ok